agent/util-scripts/pbench-tool-meister-client

```diff
@@ -16,6 +16,18 @@ from pbench.agent.constants import (
     def_redis_port,
 )
 from pbench.agent.tool_meister_client import Client
+from pbench.agent.utils import BaseServer
+
+
+class RedisServer(BaseServer):
+    """RedisServer - an encapsulation of the handling of the Redis server
+    specification.
+    """
+
+    def_port = def_redis_port
+    bad_port_ret_code = 1
+    bad_host_ret_code = 1
+    name = "Redis server"
```

Comment:
> Why isn't the definition of `RedisServer` common and shared with `pbench-tool-meister-start` and `-stop`?

Basically because it _seemed_ it would result in more code for very little gain.

At the time I did not want to embark on solving the return codes for `-stop` and `-client` while doing this work, but just wanted the shared constructor behavior.

Thank you for prompting me to look at this a bit more, as I think we can solve this elegantly.